lms/djangoapps/certificates/migrations/0018_auto__add_certificatehtmlviewconfiguration.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'CertificateHtmlViewConfiguration'
+        db.create_table('certificates_certificatehtmlviewconfiguration', (
+            ('id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('change_date', self.gf('django.db.models.fields.DateTimeField')(auto_now_add=True, blank=True)),
+            ('changed_by', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['auth.User'], null=True, on_delete=models.PROTECT)),
+            ('enabled', self.gf('django.db.models.fields.BooleanField')(default=False)),
+            ('configuration', self.gf('django.db.models.fields.TextField')()),
+        ))
```

Comment:
Just as a follow-up to this mini-thread -- we are now also providing support in this PR for course-specific overrides of certain parameters, mainly for references such as the stylesheet, logo, things like that.